src/connect/components/folder-item/folder-item.tsx

```diff
@@ -0,0 +1,45 @@
+import { DivProps, Icon } from '@/powerhouse';
+import React from 'react';
+import { twMerge } from 'tailwind-merge';
+
+export interface FolderItemProps extends DivProps {
+    title: string;
+    onOptionsClick?: DivProps['onClick'];
+}
+
+export const FolderItem: React.FC<FolderItemProps> = ({
+    title,
+    onOptionsClick,
+    ...divProps
+}) => {
+    const containerStyles = twMerge(
+        'flex flex-row bg-gray-200 group cursor-pointer h-12 items-center rounded-lg px-2 text-gray-600 hover:text-gray-800 select-none',
+        divProps.className,
+    );
+
+    return (
+        <div {...divProps} className={containerStyles}>
+            <div className="relative flex flex-1 flex-row items-center overflow-hidden">
+                <div className="p-1">
+                    <Icon name="folder-close" size={24} />
+                </div>
```

Comment:
there's a typo here — should be `text-slate` not `text-stale`

Reply:
nice catch! fix here: [3f6f985](https://github.com/powerhouse-inc/design-system/pull/128/commits/3f6f9850fc217071a42a1280adaa6cc0bc30069c)